apps/src/netsim/NetSimVizNode.js

```diff
@@ -152,6 +233,9 @@ NetSimVizNode.prototype.tick = function (clock) {
     var randomX = 300 * Math.random() - 150;
     var randomY = 300 * Math.random() - 150;
     this.tweenToPosition(randomX, randomY, 20000, tweens.easeInOutQuad);
+  } else if (this.isForeground && this.tweens_.length > 0) {
+    this.resizeNameBox_();
+    this.resizeAddressBox_();
```

Comment:
This resize during `tick` is needed because scale tweens in particular are havoc on size-matching to the text.